src/test/java/pl/coderslab/imageManager/image/ImageFacadeTest.java

```diff
@@ -0,0 +1,88 @@
+package pl.coderslab.imageManager.image;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import pl.coderslab.imageManager.exeption.ImageSavingException;
+import pl.coderslab.imageManager.mock.MockDatabase;
+
+import javax.imageio.ImageIO;
+import javax.imageio.ImageReader;
+import javax.imageio.metadata.IIOMetadata;
+import javax.imageio.stream.ImageInputStream;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+
+class ImageFacadeTest {
+
+    private ImageFacade imageFacade;
+    private ImageRepository imageRepository;
+    private static final String ABSOLUTE_IMAGE_PATH = "src/main/webapp/";
+
+    private final Path path = Paths.get("src/main/webapp/images/0/fruits.jpeg");
+    private final File file = path.toFile();
+
+    @Autowired
+    private TestEntityManager entityManager;
+
+    @BeforeEach
+    public void setUp() {
+        imageRepository = new MockDatabase();
+        imageFacade = new ImageFacade(imageRepository);
+    }
+
+    @Test
+    public void when_saving_image_return_image() throws IOException, ImageSavingException {
```

Comment:
protip to zaczynaj od should, to dobra praktyka i stosowana konwencja, czyli np. should_save_image_to_database